docs/zynq-7000-getting-started-guide.md

```diff
@@ -0,0 +1,183 @@
+# Setting up and Running a Co-Simulation with a System Clock Example
+
+This demonstration shows how to compile and run the Co-Simulation demo of Buildroot in QEMU with a simulated device in SystemC.  **Buildroot was selected for this demo, but other Linux Kernel's can be built and deployed in QEMU and used as well.** This configuration is tested working for Ubuntu 18.0.4 and assumes that a `cosim` directory is created in your home directory. This walkthrough also assumes that the device being emulated by QEMU is the Xilinx Zynq-7000 SoC.  This SoC seemed like a good candidate but the concept can apply to any QEMU machine which plugs in a compatible remoteport bus interface.
+
+## Dependencies
+
+Below are the dependencies needed to compile all the libraries in this demo:
+
+```bash
+sudo apt update
+sudo apt install cmake gmake gcc qemu-kvm qemu-system qemu-user-static verilator
+```
+
+## Setup and Compilation
+
+Run these commands to clone and build the necessary repos (`~/cosim` assumed as the base directory).
+
+### Create the base directory
+
+```bash
+mkdir ~/cosim
+```
+
+### SystemC Setup
+
+```bash
+# Build and install the binaries
+cd ~/cosim
+SYSC_VERSION=systemc-2.3.2
+wget https://www.accellera.org/images/downloads/standards/systemc/systemc-2.3.2.tar.gz
+tar xf systemc.2.3.2.tar.gz
+cd systemc-2.3.2/
+CXXFLAGS=-std=c++11 ./configure ; make
+sudo make install
+```
+
+### QEMU Setup
+
+```bash
+cd ~/cosim
+git clone https://github.com/Xilinx/qemu.git
+cd qemu
+git checkout 7e3e3ae09af8abe70383a5
+git submodule update --init dtc
```

Comment:
We can remove above two lines (commit 7e3e3ae09af8abe70383a5 could not instantiate one of the nodes in the dtb but master worked fine, and the dtc submodule is cloned automatically when building). An alternative in case we would like to fixate to a specific QEMU commit is to fixate to the current master commit.